### PR TITLE
Check for Free usage on the JSON-lD prices

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#43595](https://central.tri.be/issues/43595)

### CODE REVIEW

File: src/Tribe/JSON_LD/Order.php [hunk 1]
```diff
@@ -101,10 +101,17 @@ public function add_ticket_data( $data, $args, $post ) {
 	 * @return object
 	 */
 	public function get_offer( $ticket, $event ) {
+		$price = $ticket->price;
+
+		// JSON-LD can't have free as a price
+		if ( strpos( strtolower( trim( $price ) ), 'free' ) !== false ) {
```
Comment (borkweb) [inline src/Tribe/JSON_LD/Order.php]: In the-events-calendar, the `Free` value is set via `__( 'Free', 'the-events-calendar' )`. Should this string be translated as well?

Comment (bordoni) [inline src/Tribe/JSON_LD/Order.php]: Good Note. 😄

**Comment by bordoni** [comment]
Updated to reflect the changes suggested by @borkweb

**Comment by borkweb** [comment]
:+1:

**Pull request merged**